ci/npm-publish.yml

```diff
@@ -42,6 +39,7 @@ jobs:
           node-version: 12
           registry-url: https://npm.pkg.github.com/
           scope: '@your-github-username'
+      - run: npm ci
```

Comment:
Why is this necessary?

Reply:
`node_modules` was absent if I didn't run `npm install`.

Reply:
It is absent but does it fail when you publish?

Reply:
I did run into errors when I tried to set Actions up for two of my projects.
Has `setup-node` been changed? It could be fixed by changing somewhere else.
I ran into a 500 error when I tried to publish to GPR but now it seems to be fixed.

Reply:
BTW `npm publish` would run `npm prepare` which may require `node_modules` or whatever processes a user defines.

Reply:
I just checked my Action logs, it failed at `npm prepare` where `webpack` tried to create a package build.

Reply:
make sense then but I don't see it necessary for an easy setup - not every package has a prepare script

Reply:
I get your point, adding `npm ci` by default may generate unnecessary data transfer. Do you want me to remove this line?
And I think it's worthwhile to be added to docs, it's probably a common hurdle people may run into.